src/library_little_endian_heap.js

```diff
@@ -0,0 +1,74 @@
+/*  Stores value into the heap enforcing LE byte order. Uses DataView methods
```

Comment:
Apologies for the confusion here. This is almost what @juj was getting at, but not quite, and this is a tricky area given the history here. This is what remains to be done:

1. This file must be structured like the other `library*.js` files, which do a `mergeInto` the main library. See any of those files for an example.
2. There is then no need to `#include` the files in preamble etc. The functions will be included automatically.
3. However, the automatic system needs to be told about them since we only add them in JS post-processing. `emcc.py` must add their names to `DEFAULT_LIBRARY_FUNCS_TO_INCLUDE`. Look for `shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE +=` in `emcc.py` to see examples of that.

Reply:
Changed structure to be similar to other libraries